build.sbt

```diff
@@ -4,12 +4,13 @@ name := "tags-thrift-schema"
 organization := "com.gu"
 
 libraryDependencies ++= Seq(
-  "org.apache.thrift" % "libthrift" % "0.9.3",
+  "org.apache.thrift" % "libthrift" % "0.9.2",
```

Comment:
Why the downgrade?

Reply:
because we are using 0.9.2 throughout for compatibility with scrooge 0.4.12 and also they changed the signature of `equals` somewhere

Reply:
Ah, ok!

Will the change to the equals signature affect users of `tags-thrift-schema` or is it purely internal?

Reply:
yeah, essentially using 0.9.3 everywhere we use scrooge leads to runtime errors

Reply:
Ok no worries, should probably change the build to be `2.0.0` to reflect the fact it's a non-backwards compatible API change (from the perspective of `tags-thrift-schema`)